portexpander_i2c_bridge:
  ros__parameters:
    i2c_device: "/dev/i2c-2"
    mcp_addr: "0x20"